spoon

```diff
@@ -104,10 +104,24 @@ else
     ORIGINAL_IFS=$IFS
     IFS="^"
     select name in $NODE_NAMES; do : ; break; done
+
+    ALL_IPS=$(printf "%s\n" $FILTERED | sed 's/__/  /g' | awk '{print $3}' | xargs)
+    IFS=" " read -r -a ALL_IPS <<< "${ALL_IPS}"
     IFS=$ORIGINAL_IFS
 
     if [[ $REPLY == '*' ]]; then
       TARGET_IPS=$(printf "%s\n" $FILTERED | sed 's/__/  /g' | awk '{print $3}' | xargs)
```

Comment:
This can be refactored to use `ALL_IPS`. Should I?

Reply:
Not sure what you mean. Go ahead and do the change in one commit, I can look at it, then we can decide, and either leave it in or revert.